src/errors/ConflictError.js

```diff
@@ -1,7 +1,7 @@
 class ConflictError extends Error {
-  constructor (details) {
+  constructor (message) {
     super();
-    this.details = details;
+    this.message = message;
   }
 }
```

Comment:
Conversei com o TL, e ele comentou que faz sentido ter os dois, e o que vai mudar vai ser a necessidade de cada erro. Vai ter erro que tem details, e erro que não.

E aí ficaria algo tipo assim:
class InvalidDataError extends Error {
  constructor (details) {
    super('Corpo da requisição inválido!');
    this.details = details;
  }
}
class ForbiddenError extends Error {
  constructor () {
    super('Usuário não autorizado!');
  }
}

O mesmo vale para os demais arquivos de erro.

Reply:
É uma boa solução, se for assim vai ter que mudar um monte de coisa, back e front.